learn/03_agile_lean.md

```diff
@@ -9,25 +9,50 @@ Week Goal: To deepen your understanding of agile and lean principles and how the
 
 ### Day 1:
 * Requirements Discovery
+  * http://requirementsdiscoverycanvas.com/post/the-requirement-discovery-canvas
 * Estimation & Planning
+  * Planning poker
+  * Dot voting
+  * T-Shirt sizes
 * Retrospectives
+  * Safety
+  * Facilitation
+  * Objectives
+  * Methods
+  * Games/workshops
+  * Action points
+* Plan upcoming week using one of the techniques covered
+* Greenfield
 
 ### Day 2:
 * Kanban
+  * Discussion, highlight how this applies to what's already been done so far
+  * Encourage members to review trello and goals based on this
 * Scrum
+  * http://www.theagilemindset.co.uk/2009/06/23/the-ballpoint-game-with-a-twist/
+  * Roles
 * Waterfall
+  * Discussion of waterfall, examples, pitfalls
+  * Planning game demonstration waterfall in practice
+* Greenfield work
 
 ### Day 3:
 * Lean
+  * http://www.leansimulations.org/2010/11/penny-game.html
 * Ceremonies
-* Facilitation
+  * Cover different ceremony types
+    * Sprint planning
```

Comment:
A wider thing, but we should possibly stop using Scrum terminology such as 'Sprint' as we're, AFAIK, far from practising Scrum on any of our teams at present